Frends.Json/Frends.Json.csproj

```diff
@@ -1,10 +1,10 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0;net452</TargetFrameworks>
 	<GenerateDocumentationFile>true</GenerateDocumentationFile>
     <Description>FRENDS Json tasks</Description>
-    <Version>1.0.0.0</Version>
+    <Version>1.0.1</Version>
```

Comment:
Please don't change the version number of older official Tasks. The version will handled by the build server.